.github/workflows/ci.yml

```diff
@@ -16,9 +16,32 @@ defaults:
     working-directory: "xanmanning.k3s"
 
 jobs:
+  ansible-lint:
+    name: Ansible Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout codebase
+        uses: actions/checkout@v2
+        with:
+          path: "xanmanning.k3s"
+
+      - name: Set up Python 3
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.x"
+
+      - name: Install test dependencies
+        run: pip3 install -r molecule/lint-requirements.txt
+
+      - name: Run yamllint
+        run: yamllint -s .
+
+      - name: Run ansible-lint
+        run: ansible-lint --exclude molecule/ --exclude meta/
+
   molecule:
     name: Molecule
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
I think 20.04 will be deprecated by github at some point, was there a reason why ubuntu-latest didn't work?

Reply:
Maybe, as will all things 😄 I can't see any indication that `ubuntu-20.04` is being deprecated any time soon in https://github.com/actions/runner-images

Using `ubuntu-latest` is `22.04` and was causing Molecule tests to fail. Issues with `systemd` services not starting  (mentioned in the issue summary). I suspect it's due to Ubuntu 22.04 using cgroups v2 and some molecule config needs changing. My goal was to get the tests working as they were before.

Reply:
Sorry I was wrong it is 18.04 that got deprecated.

Reply:
So I'm looking into this a bit more @onedr0p - As I've ran into these issues with cgroups v2 on the GitHub runner distro causing issues with containers needing `systemd` 

I recall the maintainer of https://github.com/artis3n/ansible-role-tailscale experiencing similar problems. I've seen that for certain distros a certain entrypoint command is needed: https://github.com/artis3n/ansible-role-tailscale/blob/main/.github/workflows/pull_request_target.yml#L23

Reply:
https://github.com/geerlingguy/docker-ubuntu2204-ansible/issues/9#issuecomment-1489287439 For more context.